plugins/inputs/bind/bind_test.go

```diff
@@ -0,0 +1,200 @@
+package bind
+
+import (
+	"net/http"
+	"net/http/httptest"
+	"testing"
+
+	"github.com/influxdata/telegraf/testutil"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func TestBindJsonStats(t *testing.T) {
+	ts := httptest.NewServer(http.FileServer(http.Dir("testdata")))
+	defer ts.Close()
+
+	b := Bind{
+		Urls: []string{ts.URL + "/json/v1"},
```

Comment:
I added the comment based on [this testdata](https://github.com/influxdata/telegraf/pull/2383/files#diff-3756bf50200ed06a72cf33175c56b6b0R9) that I thought would exercise [this code](https://github.com/influxdata/telegraf/pull/2383/files#diff-ccaf604cd10b6bc356c51c5275de7388R105), did I miss read it?  I was thinking we just set these options to true during tests and get a few lines more coverage without much work.